bazel/dependency_imports.bzl

```diff
@@ -11,7 +11,7 @@ load("@proxy_wasm_rust_sdk//bazel:dependencies.bzl", "proxy_wasm_rust_sdk_depend
 load("@rules_cc//cc:repositories.bzl", "rules_cc_dependencies", "rules_cc_toolchains")
 
 # go version for rules_go
-GO_VERSION = "1.15.5"
+GO_VERSION = "host"
```

Comment:
Can we add a one line doc comment to explain the rationale for this change?

Reply:
This is a port from https://github.com/maistra/proxy/blob/maistra-2.1/maistra/scripts/update-deps.sh#L97 - I removed that patch from proxy in 2.2.
We use the host's golang (as we use java, gcc, etc from the host)